_articles/what-is-oauth-understanding-the-authorization-layer.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: article
+title: "What is OAuth? Understanding the authorization layer"
+abstract: "A simple explanation of what OAuth is, how it works and what it is used for."
+sitemap:
+  lastmod: 2020-05-15
+---
+
+OAuth is short for Open Authorization.  OAuth is a framework for token-based authorization on the internet. What does that mean?  Basically, OAuth is a way for websites to share private information with each other. Authorization tokens are issued which grants access to specific pieces of information hence the term token-based authorization.
+
+For example, let’s say you want your social media profile to share with a dating website your name, date of birth and height.  OAuth can be used to authorize the dating app access to those details on your social media profile.  The OAuth framework allows the social media profile to create an access token with certain attributes, such as an expiry date, which the dating website can use to access the data the token has scope for.
+
+This method allows websites to share only the specific pieces of information it chooses with other websites.  Before OAuth was invented, if two websites wanted to share information they had little choice but to share everything at an account level:
+
+> “Third-party applications gain overly broad access to the resource owner's protected resources, leaving resource owners without any ability to restrict duration or access to a limited subset of resources.”
+
+[Source - The OAuth 2.0 Authorization Framework By the Internet Engineering Task Force](https://tools.ietf.org/html/rfc6749)
+
+OAuth is all about granting authorization to access private things.  OAuth is not used for authentication.  A mistake commonly made is to assume the ‘auth’ in OAuth stands for authentication.
```

Comment:
DID.app is all about authentication, theres probably a comment that Authentication can be built on top of OAuth, and how to do this is standardised by specifications like OpenID Connect

Reply:
Totally.  I come onto this later in the article...